content/en/docs/concepts/security/index.md

```diff
@@ -632,331 +628,260 @@ authorization result, `ALLOW` or `DENY`.
-objects, `ClusterRbacConfig` is defined as a
-Kubernetes `CustomResourceDefinition`
-[(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) object.
-
-In the `ClusterRbacConfig` object, the operator can specify a `mode` value, which can
-be:
-
-- **`OFF`**: Istio authorization is disabled.
-- **`ON`**: Istio authorization is enabled for all services in the mesh.
-- **`ON_WITH_INCLUSION`**: Istio authorization is enabled only for services and
-  namespaces specified in the `inclusion` field.
-- **`ON_WITH_EXCLUSION`**: Istio authorization is enabled for all services in
-  the mesh except the services and namespaces specified in the `exclusion`
-  field.
+You enable Istio Authorization on **workloads** by applying the `AuthorizationPolicy`
+custom resource that selects the workloads. Currently `AuthorizationPolicy` only
+supports "ALLOW" action. This means that if multiple authorization policies apply
+to the same workload, the effect is additive.
 
-In the following example, Istio authorization is enabled for the `default`
-namespace.
+The following authorization policy enables access control on workloads with labels
+`app: httpbin` and `version: v1` in namespace foo.
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ClusterRbacConfig
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: default
+ name: httpbin
+ namespace: foo
 spec:
-  mode: 'ON_WITH_INCLUSION'
-  inclusion:
-    namespaces: ["default"]
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep"]
+   to:
+   - operation:
+       methods: ["GET"]
+   when:
+   - key: request.auth.claims[iss]
+     values: ["https://accounts.google.com"]
 {{< /text >}}
 
 ### Authorization policy
 
-To configure an Istio authorization policy, you specify a `ServiceRole` and
-`ServiceRoleBinding`. Like other Istio configuration objects, they are
-defined as
-Kubernetes `CustomResourceDefinition` [(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) objects.
+To configure an Istio authorization policy, you specify the [Authorization Policy](https://github.com/istio/api/blob/bfa91e88abf1af16d248917ebd3ab62036f7ca12/security/v1beta1/authorization.proto)
+configuration object. Like other Istio configuration objects, it is defined as Kubernetes `CustomResourceDefinition` [(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+object.
 
-- **`ServiceRole`** defines a group of permissions to access services.
-- **`ServiceRoleBinding`** grants a `ServiceRole` to particular subjects, such
-  as a user, a group, or a service.
+An `AuthorizationPolicy` includes a `selector` and a list of `rules`, the `selector` specifies
+**where** to apply the policy and the `rules` specify **who** is allowed to do **what** under which **conditions**.
```

Comment:
specifies **where** to apply the policy => specifies the **target" that the policy applies to.